sqlstats/sqlstats.go

```diff
@@ -70,14 +83,14 @@ func (t *Tracer) done(s *connStats) (why string, readOnly bool) {
 	return why, readOnly
 }
 
-type queryStats struct {
-	query string
+type QueryStats struct {
+	Query string
 
 	// When inside the queries map all fields must be accessed as atomics.
-	count    int64
-	errors   int64
-	duration int64 // time.Duration
-	mean     int64
+	Count    int64
+	Errors   int64
+	Duration int64 // time.Duration
```

Comment:
Does the comment mean we should convert it into a `time.Duration` or that the units are nanoseconds? (Or both?) Unless the extra cost of conversion is a performance consideration, I'd suggest making it a duration anyway—and if we don't want to do that for some reason we should probably put the units in the name, e.g., `DurationMicros` or something.